DataProducts/inc/CaloSiPMId.hh

```diff
@@ -0,0 +1,41 @@
+#ifndef DataProducts_CaloSiPMId_hh
+#define DataProducts_CaloSiPMId_hh
+//
+// Offline identifier of one calorimeter SiPM channel
+// Online numbering is in CaloRawSiPMId
+//
+
+#include "Offline/DataProducts/inc/CaloConst.hh"
+#include "Offline/DataProducts/inc/CrystalId.hh"
+#include <iosfwd>
+
+namespace mu2e {
+
+  class CaloSiPMId{
+
+  public:
+
+    using value_type = CaloConst::CaloSiPMId_type;
+
+    explicit CaloSiPMId(value_type id) { _id = id; }
+
+    value_type id() const { return _id; }
+    value_type SiPM01() const { return _id%CaloConst::_nSiPMPerCrystal; }
```

Comment:
What does SiPM01 means?

Reply:
If you just say SiPM, it is not clear if you mean the SiPM number 0-2500 (number in the whole system) or 0-1 (number in this crystal)  The "01" is supposed to indicate the latter.  One could also imagine SiPMinCrystal or something like that.  Make a suggestion if you would like a different name.